docs/source/getting-started.rst

```diff
@@ -49,10 +49,18 @@ The easiest way to use Glow in the cloud is with the `Databricks Runtime for Gen
 provider or Spark distribution. You need to install the maven package
 ``io.project:glow_2.11:${version}`` and optionally the Python frontend ``glow.py``.
 
-Example notebook
-----------------
+Notebooks embedded in the docs
```

Comment:
I added a similar section for this here: https://github.com/projectglow/glow/blob/master/docs/source/additional-resources.rst (there is also a hard-coded link to this in the notebook extension: https://github.com/projectglow/glow/blob/master/docs/extensions/notebook.py). We should probably choose which of these to point to. Did we remove any reference to `dbutils` as well?

Reply:
edited and made a reference to your paragraph